src/Components/Header/Header.js

```diff
@@ -26,16 +34,19 @@ if (path.includes('account')) {
   return (
     <header className='main-header'>
       <section className='zip-code-section'>
-        {zipCode &&
+        {zipCode && 
         <>
         <p>Zip Code: {zipCode}</p>
         {!selectedMarketId && <button className='zip-code-reset-button' onClick={() => dispatch(addZipCode(''))}>Reset Zip Code</button>}
-        </>}
+          </>}
+        {vendors.length > 0 &&
```

Comment:
We already talked about this, but I think we might need to find another conditional to render this button on since we are using the same state of Vendors when we are on the Vendor side of the app to store all vendors. Maybe we can use the path to render it on that specific page